src/plugins/simulator/components/simulator-tile.scss

```diff
@@ -7,15 +7,15 @@
   overflow: auto;
 
   .simulator-content {
-    height: 100%;
+    height: calc(100% - 45px); // The 45 pixels is to accommodate the tile title
     width: 100%;
 
     .simulator-variables {
       display: flex;
       font-family: 'Ubunutu', sans-serif;
```

Comment:
Not part of your changes, but I'm pretty sure this is a typo for 'Ubuntu' font.

Reply:
Whoops! Good catch.